routes/register.go

```diff
@@ -395,6 +395,7 @@ func Register(c Config) {
 					r.Use(ctx.RequireViewDataset)
 
 					// view only functions
+					r.With(ctx.SetNav("datasets"), ctx.SetSubNav("contributors")).Get("/contributors", datasetviewing.ShowContributors).Name("dataset_contributors")
```

Comment:
SetNav not needed, already set higher up

Reply:
Ah where?